sql/moz-fx-data-shared-prod/experimenter_backend_derived/data_collection_opt_out_events_v1/query.sql

```diff
@@ -0,0 +1,13 @@
+SELECT
+  submission_timestamp,
+  _extra.value AS nimbus_user_id,
+FROM
+  `moz-fx-data-shared-prod.experimenter_backend.events`,
```

Comment:
I think `moz-fx-data-shared-prod.experimenter_backend.events_stream` would work as well and would avoid the unnests.  That would be more efficient if there are a lot of events